src/color.js

```diff
@@ -110,7 +110,7 @@ module.exports = class Color {
       color = Color.parse(color);
     }
 
-    if (color == null) {
+    if (color === null) {
```

Comment:
this is also a behavioural change. The previous `==` was catching both null and undefined.
This will not catch an undefined, moving forward with the code and trying to access properties of an undefined.

Reply:
color will always be either null or an instance of Color

Reply:
to me seems that in this case color is an argument of the function, so can be whatever people pass down. Calling the method with no argument before would return false, now will throw an error. Right?

As long as this change is acknowledged and willingful, that is fine.

Reply:
Seems like throwing an error would be the correct thing to do anyway if `color.isEqual` is called without an argument.

Reply:
Lines 109 to 111 say
```javascript
    if (!(color instanceof Color)) {
      color = Color.parse(color);
    }
```
So actually, `color` has to be `instanceof Color | ReturnType<Color.parse>`
And `Color.parse` never returns `undefined`, therefore `color` can never be `undefined` either

Reply:
oh you are right i totally missed that.